### PR TITLE
UGENE-6983. Support passing multiple test labels to gui-test-launcher.

### PR DESCRIPTION
**Pull request by yalgaer**

### CODE REVIEW

File: src/corelibs/U2Core/src/util/TextUtils.cpp [hunk 1]
```diff
@@ -214,4 +214,16 @@ QStringList TextUtils::split(const QString &text, int chunkSize) {
...
+bool TextUtils::checkFirstListContainsSecond(const QStringList &firstList, const QStringList &secondList) {
+    if (firstList.size() < secondList.size()) {
+        return false;
+    }
+    for (const QString &valueFromSecondList : secondList) {
+        if (!firstList.contains(valueFromSecondList)) {
+            return false;
+        }
+    }
+    return true;
```
Comment (atiunov) [inline src/corelibs/U2Core/src/util/TextUtils.cpp]: You can use ```firstList.toSet().contains(secondList.toSet())```
But you can leave this method alone, if you like.

File: src/corelibs/U2Test/src/gui_tests/GUITestLauncher.cpp [hunk 2]
```diff
@@ -161,28 +161,31 @@ bool GUITestLauncher::initTestList() {
     UGUITestBase *guiTestBase = AppContext::getGUITestBase();
     SAFE_POINT(guiTestBase != nullptr, "Test base is NULL", false);
 
+    // Tests labels sed to build a test set.
```
Comment (atiunov) [inline src/corelibs/U2Test/src/gui_tests/GUITestLauncher.cpp]: "sed"?

File: src/corelibs/U2Test/src/gui_tests/GUITestLauncher.cpp [hunk 3]
```diff
@@ -229,7 +232,7 @@ bool GUITestLauncher::initTestList() {
     } else {
         // Run all tests with the given label as a single suite.
         // If label is not provided: all tests are selected.
-        testList = guiTestBase->getTests(UGUITestBase::Normal, qgetenv("UGENE_GUI_TEST_LABEL"));
+        testList = guiTestBase->getTests(UGUITestBase::Normal, labelList);
```
Comment (atiunov) [inline src/corelibs/U2Test/src/gui_tests/GUITestLauncher.cpp]: Fix the comment. As I understand, tests that contain all the labels in the list will be run.

**Review by atiunov — CHANGES_REQUESTED** [review]

File: src/corelibs/U2Test/src/gui_tests/GUITestLauncher.cpp (hunk 2 above)
Comment (yalgaer) [reply src/corelibs/U2Test/src/gui_tests/GUITestLauncher.cpp]: Done

File: src/corelibs/U2Test/src/gui_tests/GUITestLauncher.cpp (hunk 3 above)
Comment (yalgaer) [reply src/corelibs/U2Test/src/gui_tests/GUITestLauncher.cpp]: Done

File: src/corelibs/U2Core/src/util/TextUtils.cpp (hunk 1 above)
Comment (yalgaer) [reply src/corelibs/U2Core/src/util/TextUtils.cpp]: I removed this method completely and made GUITest use QSet<QString> for labels.

**Review by atiunov — APPROVED** [review]

**Pull request merged**